fancy_cache/memory.py

```diff
@@ -1,12 +1,20 @@
+import logging
 import re
+import typing
 
 from django.core.cache import cache
 
-from fancy_cache.middleware import REMEMBERED_URLS_KEY, LONG_TIME
+from fancy_cache.middleware import (
+    REMEMBERED_URLS_KEY,
+    LONG_TIME,
+    USE_MEMCACHED_CAS,
+)
 from fancy_cache.utils import md5
 
 __all__ = ("find_urls",)
 
+LOGGER = logging.getLogger(__name__)
```

Comment:
This seems like an unrelated change. 
If it's the right thing to have logging, it should be its own PR.